app/root.tsx

```diff
@@ -90,10 +91,15 @@ function App() {
         <Links />
       </head>
       <body
-        // TODO: switch this to clsx when Stephan's PR that adds it is merged
-        className={[showPendingState ? 'opacity-50' : null, theme]
-          .filter(Boolean)
-          .join(' ')}
+        className={clsx(
+          'transition',
```

Comment:
Theme stuff is kinda wonky because we need to server render the right theme, but we don't know what the right theme is on the server. I'll look into this more 👍